src/test/scala/org/ergoplatform/mining/ErgoMinerSpec.scala

```diff
@@ -98,10 +99,7 @@ class ErgoMinerSpec extends FlatSpec with ErgoTestHelpers with ValidBlocksGenera
           r.s.stateContext).get
       }
 
-      // put txs in mempool altogether to speedup test
-      await(nodeViewHolderRef ? GetDataFromCurrentView[ErgoHistory, UtxoState, ErgoWallet, ErgoMemPool, Unit] { v =>
-        v.pool.put(txs)
-      })
+      txs.foreach(nodeViewHolderRef ! LocallyGeneratedTransaction(_))
```

Comment:
@catena2w, Yes, it will make more logs, but there was no significant performance deterioration observed. We are no longer able to put txs directly to pool, as it is immutable and has to be updated inside NHV.